kernel/comps/virtio/src/device/network/device.rs

```diff
@@ -115,11 +116,11 @@ impl NetworkDevice {
             .unwrap();
         device
             .transport
-            .register_queue_callback(QUEUE_SEND, Box::new(handle_send_event), false)
+            .register_queue_callback(QUEUE_SEND, Box::new(handle_send_event), true)
             .unwrap();
         device
             .transport
-            .register_queue_callback(QUEUE_RECV, Box::new(handle_recv_event), false)
+            .register_queue_callback(QUEUE_RECV, Box::new(handle_recv_event), true)
```

Comment:
Does this change (using standalone IRQs for sending/receiving queues) affect the performance?

Reply:
Yes, it has a few effects on the performance. Below are the results on CI machine. (But the results are not consistent on my local machine, in my local test, setting unique interrupt contributes the most to the performance.

  | Asterinas(MB/sec) 
-- | -- 
Before optimized | 232 
Only set unique interrupt | 239 
Optimized | 255

Reply:
I checked the Linux settings and think the changes here are reasonable:
```
~ # ls -al sys/devices/pci0000:00/0000:00:02.0/virtio0/driver -al
lrwxrwxrwx    1 root     root             0 Nov 11 10:36 sys/devices/pci0000:00/0000:00:02.0/virtio0/driver -> ../../../../bus/virtio/drivers/virtio_net
~ # cat proc/interrupts | grep 'virtio0'
 26:          0   PCI-MSI 32768-edge      virtio0-config
 27:          0   PCI-MSI 32769-edge      virtio0-input.0
 28:          0   PCI-MSI 32770-edge      virtio0-output.0
```